com.unity.multiplayer.mlapi/Runtime/Core/NetworkObject.cs

```diff
@@ -218,7 +224,7 @@ public static void NetworkShow(List<NetworkObject> networkObjects, ulong clientI
         {
             if (networkObjects == null || networkObjects.Count == 0)
             {
-                throw new ArgumentNullException("At least one NetworkObject has to be provided");
+                throw new ArgumentNullException("At least one " + nameof(NetworkObject) + " has to be provided");
```

Comment:
string interpolation is more performant and more modern than string concatenation.

```suggestion
                throw new ArgumentNullException($"At least one {nameof(NetworkObject)} has to be provided");
```